.Rbuildignore

```diff
@@ -53,3 +53,4 @@ vignettes/extra
 vignettes/build
 vignettes/mrgsolve-builds
 vignettes/extra/mrgsolve-builds/
+LICENSE
```

Comment:
Thanks, @kyleam. Main motivation here is to get the license registered with github. I nuked the `LICENSE` file and dropped that from `.Rbuildignore`. Then, ran the `usethis` command which added `LICENSE.md` with appropriate ignore. I believe this will still be recognized by [github](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).